Converters/UpdateConverterInstagram.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Buffer\UpdateConverters;
+
+use MongoDB\BSON\UTCDateTime;
+
+class UpdateConverterInstagram implements NativeUpdateConverter
+{
+    public static function convertFromSocialNetwork($post)
+    {
+        $update_attrs = [
```

Comment:
Not a biggie but would love if we can keep PSR-2 coding standard for all our PHP libraries including this one. The variables should be camel case. https://www.php-fig.org/psr/psr-2/

Reply:
If you're using VS Code then installing php_codesniffer [extension](https://marketplace.visualstudio.com/items?itemName=ikappas.phpcs) is so useful :)

Reply:
Eek! Good catch! I was missing it on VSCode! 😄